etc/ReplayOfflineConfiguration.py

```diff
@@ -106,18 +109,19 @@
 
 # Defaults for CMSSW version
 defaultCMSSWVersion = {
-    'default': "CMSSW_14_0_1"
+    'default': "CMSSW_14_0_2"
 }
 
 # Configure ScramArch
 setDefaultScramArch(tier0Config, "el8_amd64_gcc11")
 setScramArch(tier0Config, "CMSSW_12_4_9", "el8_amd64_gcc10")
 setScramArch(tier0Config, "CMSSW_12_3_0", "cs8_amd64_gcc10")
 setScramArch(tier0Config, "CMSSW_14_0_1", "el8_amd64_gcc12")
+setScramArch(tier0Config, "CMSSW_14_0_2", "el8_amd64_gcc12")
 
 
 # Configure scenarios
-ppScenario = "ppEra_Run3_2023"
+ppScenario = "ppEra_Run3"
```

Comment:
@malbouis @jordan-martins

as per https://github.com/cms-sw/cmssw/issues/43756#issuecomment-1923369576 I think we should move back to `ppEra_Run3` year, right?

Reply:
Thanks, @mmusich . Yes, I would say so, since 'ppEra_Run3' is the one without the customization for the hcal thresholds.